src/sentry/runner/commands/devserver.py

```diff
@@ -31,6 +31,7 @@
         "latest",
     ],
     "metrics": ["sentry", "run", "ingest-metrics-consumer-2"],
+    "profiles": ["sentry", "run", "ingest-profiles"],
```

Comment:
I don't think we want to enable this by default yet for everyone

Reply:
Could use a `--profiling` flag in devserver to enable it?